examples/playground/json-examples/carto-bq-tiler.json

```diff
@@ -0,0 +1,26 @@
+{
+  "description": "CartoBQTilerLayer declarative example",
+  "initialViewState": {
+    "latitude": 40.7368521,
+    "longitude": -73.8536065,
+    "zoom": 9,
+    "pitch": 0,
+    "bearing": 0
+  },
+  "views": [
+    {
+      "@@type": "MapView",
+      "controller": true,
+      "mapStyle": "https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
+    }
+  ],
+  "layers": [
+    {
+      "@@type": "CartoBQTilerLayer",
+      "data": "cartobq.maps.nyc_taxi_points_demo_id",
+      "getFillColor": [232, 133, 113],
```

Comment:
I'd like to have a function to style by color using a property. Is this something feasible?

Something like this:
```javascript
getFillColor: (object) => {
  // Apply color based on an attribute
  if (object.properties.aggregated_total > 1000000) {
    return [255, 0,0];

  } else {
    return [0, 0, 255] ;
  }
}
```

Any ideas?

Reply:
See https://deck.gl/docs/api-reference/json/conversion-reference#functions-and-using-the--prefix